error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -782,6 +782,7 @@ Flux<S> after(Flux<T> flux) {
   /** Prefer {@link Mono#flatMap(Function)} over more contrived alternatives. */
   static final class MonoFlatMap<S, T> {
     @BeforeTemplate
+    @SuppressWarnings("MonoOfPublishers")
```

Comment:
Does this mean we do not need these Refaster rules, since bug checker will warn this statement ?

Reply:
Well yes there is overlap, but the `MonoOfPublishers` doesn't do autofixes. So if we suppress these rules, than some of the violations are auto-fixed by Refaster, which is nice 😄. Generally if there is overlap and the check comes with `SuggestedFix`es, we'd drop the Refaster rule./

Reply:
Makes sense 💯  😄